presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -901,6 +902,36 @@ public static double chiSquaredCdf(
         return distribution.cumulativeProbability(value);
     }
 
+    @Description("inverse of Gamma cdf given shape and scale parameter and probability")
+    @ScalarFunction
+    @SqlType(StandardTypes.DOUBLE)
+    public static double inverseGammaCdf(
+            @SqlType(StandardTypes.DOUBLE) double shape,
+            @SqlType(StandardTypes.DOUBLE) double scale,
+            @SqlType(StandardTypes.DOUBLE) double p)
+    {
+        checkCondition(p >= 0 && p <= 1, INVALID_FUNCTION_ARGUMENT, "p must be in the interval [0, 1]");
+        checkCondition(shape > 0, INVALID_FUNCTION_ARGUMENT, "shape must be greater than 0");
+        checkCondition(scale > 0, INVALID_FUNCTION_ARGUMENT, "scale must be greater than 0");
+        GammaDistribution distribution = new GammaDistribution(null, shape, scale, GammaDistribution.DEFAULT_INVERSE_ABSOLUTE_ACCURACY);
```

Comment:
Will there be any need of changing the accuracy when using this function? If yes, can make this a function argument too.
A total noob on math here, I see that the default accuracy is 1e-9, which I guess should be enough. Just a reminder if as a user you foresee such case.

Reply:
Thanks @feilong-liu 

I doubt such a need would arise.
If it does, then it might be worth adding that argument to all the probability functions (there are a bunch).
I'm going to leave it as is for now.